### PR TITLE
Ignoring more generated file for Java project

### PR DESCRIPTION
**Pull request by tcthien**
**Reasons for making this change:**

- Adding generated Eclipse project to ignore
- Adding log file, zip, tar.gz, rar to ignore
- Adding generated vagrant file to ignore

### CODE REVIEW

File: Java.gitignore [hunk 1]
```diff
@@ -1,5 +1,13 @@
 *.class
 
+# Eclipse editor
```
Comment (shiftkey) [inline Java.gitignore]: We have an existing [`Global\Eclipse.gitignore`](https://github.com/github/gitignore/blob/master/Global/Eclipse.gitignore). Please drop these rules.

File: Java.gitignore [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+.classpath
+.project
+.settings/
+
+# Log file
+.log
```
Comment (shiftkey) [inline Java.gitignore]: Should the be `*.log`?

File: Java.gitignore [hunk 3]
```diff
@@ -10,6 +18,12 @@
 *.jar
 *.war
 *.ear
+*.zip
+*.tar.gz
+*.rar
+
+# Vagrant
```
Comment (shiftkey) [inline Java.gitignore]: We have an existing [`Global\Vagrant.gitignore`](https://github.com/github/gitignore/blob/master/Global/Vagrant.gitignore). Please drop these rules.

**Comment by shiftkey** [comment]
@tcthien thanks for these changes. I've called out the ones that are stored elsewhere, but the rest seem fine.

**Comment by tcthien** [comment]
Got the idea, I wasn't aware about global ignore. I'm going to remove the duplicate and open another PR

**Pull request closed without merging**